lib/scss/inc/_aside.scss

```diff
@@ -114,35 +118,28 @@
   list-style: none;
   display: none;
 
+  .doc-link__active {
+    color: $doclink-active-color;
+  }
+
   & > li {
     & > a {
       display: block;
       transition: color 300ms;
-      padding: 5px 40px;
-      font-size: $font-size-base - 0.7;
-
-      &:not(.doc-link__active) {
-        color: $doclink-color;
-      }
+      color: $doclink-color;
+      padding: 5px 0;
+      font-size: $font-size-base - 2;
```

Comment:
Could we keep this as `$font-size-base - 0.7` if possible as per Jayshree's feedback in the previous commit?